src/utils.ts

```diff
@@ -5,6 +5,11 @@ import type {
   JsonRpcSuccess,
 } from './JsonRpcEngine';
 
+export const hasProperty = (
+  object: Object, // eslint-disable-line @typescript-eslint/ban-types
+  name: string | number | symbol,
+): boolean => Reflect.hasOwnProperty.call(object, name);
```

Comment:
`Reflect` doesn't have that method. It only works because the property access walks the prototype chain back to `Object`:
```suggestion
): boolean => Object.hasOwnProperty.call(object, name);
```